pkg/controller/gitopsconfig/controller.go

```diff
@@ -244,19 +246,19 @@ func (r *Reconciler) createJob(jobtype string, instance *gitopsv1alpha1.GitOpsCo
 	job, err := util.CreateJob(mergedata)
 	if err != nil {
 		log.Error(err, "unable to create job manifest from merge data", "mergedata", mergedata)
-		return reconcile.Result{}, err
+		return reconcile.Result{}, xerrors.Errorf("unable to create job manifest from merge data %q: %w", mergedata, err)
```

Comment:
I believe mergedata may be quite huge, so I'm not convinced we want to try printing it all here; also, what will `%q` print in this case?